README.md

```
@@ -49,6 +49,8 @@ npm i
 npm run dev
 ```
 
+If you faced problems with ```npm i ```, you may try ```npm i --force``` instead
```

Markdownlint

Spaces inside code span elements

(MD038, no-space-in-code)